@here/harp-mapview/lib/MapView.ts

```diff
@@ -2956,6 +2964,40 @@ export class MapView extends THREE.EventDispatcher {
         const worldOffsetX = tile.computeWorldOffsetX();
         if (tile.willRender(zoomLevel)) {
             for (const object of tile.objects) {
+                if (object.userData.technique && object.userData.feature) {
```

Comment:
We need to find the right place (and the right way) to get the techniques associated with the objects.